README.md

```diff
@@ -149,6 +149,14 @@ RetryConfig config = new RetryConfigBuilder()
         .build();
 ```
 
+If you want the executor to continue to retry when encountered exception's cause is among a list of exceptions, then specify **retryOnCausedBy()** config option
```

Comment:
i might clean this README blurb up a little bit to have more details but that's fine to do after merging